### ISSUE TITLE
Design Intent - Accessibility Feedback - Authenticated Experience My VA, Onsite Notifications - Badge Indicator

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#92692]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Really solid and well thought out. The only things I have are small or nitpicky. Good work all around!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Must:** For the ARIA text on the link, flip the order around: `Go to your inbox. You have # new messages.` This preserves the same reading order as the visual order of the link, and it should make voice command software more likely to get a match when someone says "Click Go to your inbox."
- [ ] **Consider:** It looks like the color contrast for white text on the red badge is [6.98:1](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=B50909). That passes WCAG AA requirements easily, but it's 0.02 short of passing AAA (7:1). Does a 0.02 difference actually matter? Probably not. But if there's a good red that's a hair darker that you can switch to and get over that 7:1 threshold, it's always great to pass that higher standard.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by ajakabcin** [comment]
Thank you @briandeconinck! we'll be reordering the aria and text and we'll see what we can do about the color but I know Matt has already tinkered with it a bunch to keep it in compliance.. We'll do our best!

**Comment by mattmarino-adhoc** [comment]
Thanks @briandeconinck!!!
